cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/partitioned/PartitionExploreCorrector.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.api.dataset.lib.partitioned;
```

Comment:
shall we put it in a separate package, e.g. `c.c.c.api.dataset.lib.app.partitioned`? I am assuming there maybe need for dataset related app in future for various kind of tasks (e.g. data copying/migration, schema conversion... etc).